From 8459b35ec0a2c1cddd7dab8b726e752bcde4c609 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B8ren=20Sandmann=20Pedersen?= <ssp@redhat.com>
Date: Sat, 1 Sep 2012 11:19:41 -0400
Subject: [PATCH 1/2] Add A8 surface capability

Even though the ability to handle a8 surfaces was added at the same
time as the composite command, they are logically separate, so add a
capability bit to indicate the presence of a8 surfaces.
---
 spice-common/spice-protocol/spice/protocol.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/spice-common/spice-protocol/spice/protocol.h b/spice-common/spice-protocol/spice/protocol.h
index 7008399..0671292 100644
--- a/spice-common/spice-protocol/spice/protocol.h
+++ b/spice-common/spice-protocol/spice/protocol.h
@@ -128,6 +128,7 @@ enum {
     SPICE_DISPLAY_CAP_SIZED_STREAM,
     SPICE_DISPLAY_CAP_MONITORS_CONFIG,
     SPICE_DISPLAY_CAP_COMPOSITE,
+    SPICE_DISPLAY_CAP_A8_SURFACE,
 };
 
 enum {
-- 
1.7.4

